ietf/doc/views_doc.py

```diff
@@ -533,6 +533,7 @@ def document_main(request, name, rev=None, document_html=False):
                                        review_assignments=review_assignments,
                                        no_review_from_teams=no_review_from_teams,
                                        due_date=due_date,
+                                       diff_revisions=get_diff_revisions(request, name, orig_doc) if document_html else None
```

Comment:
There's a pattern that occurs quite often elsewhere in the code to deal with what this is achieving. Consider using it here since 145 and 536 are so far apart:

`doc if isinstance(doc,Document) else doc.doc`

(At some future refactor we may want to make this a method on DocumentInfo)